test/fixtures/es-module-loaders/assertionless-json-import.mjs

```diff
@@ -1,17 +1,22 @@
 const DATA_URL_PATTERN = /^data:application\/json(?:[^,]*?)(;base64)?,([\s\S]*)$/;
 const JSON_URL_PATTERN = /\.json(\?[^#]*)?(#.*)?$/;
```

Comment:
Not related to this PR, but I realize that the regex would count a URL such as `file:///file.js?k=file.json` as a URL pointing to a JSON file.
```suggestion
const JSON_URL_PATTERN = /^[^?]+\.json(\?[^#]*)?(#.*)?$/;
```